README.md

```diff
@@ -228,6 +244,71 @@ ignore (default: `0`)
 - `-latency-p50`: Expected maximum latency for 50th percentile of requests, 0 to
 ignore (default: `0`)
 
+## Observability & Troubleshooting
+
+### What's happening with the rollout of my service?
+
+To view the current state of the rollout of a particular service, you can see
+the service's routing information.
+
+In Cloud Run, under the `Revisions` section, the first thing to notice is that
```

Comment:
need to mention Cloud Console probably. you can link to it as well

Reply:
I moved "In Cloud Run" to the end of the previous paragraph and added a link to [http://console.cloud.google.com/run](http://console.cloud.google.com/run)